frontend/src/components/story/CategoryChip.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-chip v-if="!category()._meta.loading"
+          :color="category().color"
+          class="px-1 tabular-nums e-category-chip"
```

Comment:
Ich mag die kleineren Clips. px-1 ist aber extrem knapp.

Gegenvorschlag:
px-1 entfernen.
`.e-category-chip` in `<style scoped>` um `padding: 1.5px 7px 0px 7px;` ergänzen

**Vorschau:**
![image](https://user-images.githubusercontent.com/470237/163791150-481663b8-caa0-4386-aa39-0ce9bfeb438e.png)